src/lib/consenthandler/consentHandler.js

```diff
@@ -41,20 +41,28 @@ export default class ConsentHandler{
      * @returns {boolean}
      */
     consentEnabled(){
-        return !!this.proxy;
+        return this.config.type === 'iab';
```

Comment:
this seems like a pretty strict check, since we got rid of the static type.  is there some other valid type?

Reply:
This is it.  It's either '' or 'iab' at the moment.